eslint.yaml

```diff
@@ -11,7 +11,7 @@ globals:
 rules:
   strict: [2, "function"]
   quotes: 0
-  camelcase: 2
+  camelcase: 1
```

Comment:
Is there a reason for this change?

Reply:
Yes, was throwing an error due to new task in gruntfile 'convert_within_custom_replacements' not being camelcase:
```
lessToSass: {
        convert_within_custom_replacements: 
```